sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -33,6 +37,8 @@
 from apache_beam.runners.worker.log_handler import FnApiLogRecordHandler
 from apache_beam.runners.worker.sdk_worker import SdkHarness
 
+standard_library.install_aliases()
```

Comment:
Do we want to call this before we do the import of "http.server" above?  It looks like this only works because of an "accident" in that this module is imported after someone else already called `install_aliases()`.

Reply:
Order should be OK, see: https://github.com/apache/beam/pull/5373#discussion_r200076690